src/Frontend/Page/Delegation.hs

```diff
@@ -73,20 +76,24 @@ instance FormPage PageDelegateVote where
                   | otherwise                         -> DF.Error "user id not found"
         valid bad = DF.Error ("corrupt form data: " <> bad ^. showed . html)
 
-    formPage v f p@(PageDelegateVote scope options _mselected) = semanticDiv p . f $ do
+    formPage v f p@(PageDelegateVote scope options mselected) = semanticDiv p . f $ do
         h1_ [class_ "main-heading"] "Stimme beauftragen"
         div_ [class_ "sub-heading"] $ do
             let delegationText name = "Wähle einen Beauftragten für " <> show name
             toHtml . delegationText $
                 either (view topicTitle) (view ideaTitle) scope
+            -- TODO: Translate
+            br_ []
+            "You can revoke your delegation unselecting the one you delegated to."
         ul_ $ do
             DF.inputHidden "selected-delegate" v
             div_ [class_ "delegate-image-select"] $ do
                 ul_ . for_ options $ \user -> do
                     let url = "avatars/" <> uid <> ".png"
                         uid = user ^. _Id . unAUID . showed
                         unm = user ^. userLogin . unUserLogin
-                    li_ [ class_ "icon-list-button col-3-12"
+                    li_ [ class_ $ "icon-list-button col-3-12"
+                                        <> maybe "" (bool "" "m-active" . (user ^. _Id ==)) mselected
```

Comment:
shouldn't be necessary, because js should do this.  don't fix it, but please add a FIXME.